libkpod/containerData.go

```diff
@@ -155,20 +171,35 @@ func inspectContainer(store storage.Store, container string) (*oci.Container, er
 	return ociCtr, nil
 }
 
+func getBlankSpec() specs.Spec {
+	return specs.Spec{
+		Process:     &specs.Process{},
+		Root:        &specs.Root{},
+		Mounts:      []specs.Mount{},
+		Hooks:       &specs.Hooks{},
+		Annotations: make(map[string]string),
+		Linux:       &specs.Linux{},
+		Solaris:     &specs.Solaris{},
+		Windows:     &specs.Windows{},
+	}
+}
+
 // get an oci.Container instance for a given container ID
 func getOCIContainer(store storage.Store, container string) (*oci.Container, error) {
 	ctr, err := FindContainer(store, container)
 	if err != nil {
 		return nil, err
 	}
+
+	m := getBlankSpec()
 	config, err := store.FromContainerDirectory(ctr.ID, "config.json")
-	if err != nil {
+	if err != nil && !os.IsNotExist(errors.Cause(err)) {
```

Comment:
If cri-o has never started the container, that file won't exist.  This lets us keep going, for example, with buildah containers.